adafruit_vs1053.py

```diff
@@ -107,12 +115,10 @@ def __init__(self, spi, cs, xdcs, dreq):
         # Check version is 4 (VS1053 ID).
         if self.version != 4:
             raise RuntimeError(
-                "Expected version 4 (VS1053) but got: {}  Check wiring!".format(
-                    self.version
-                )
+                f"Expected version 4 (VS1053) but got: {self.version}  Check wiring!"
             )
 
-    def _sci_write(self, address, value):
+    def _sci_write(self, address, value) -> None:
```

Comment:
`address` and `value` need to be `int` based on how they're used here.

Reply:
duh.....